observability/opentelemetry-extension/src/main/resources/subsystem-templates/opentelemetry.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
This file can be dropped now. We no longer produce pre-Galleon legacy feature packs, which is what used these,

Reply:
Removed.